src/pip/_internal/cache.py

```diff
@@ -168,6 +188,7 @@ def __init__(self, format_control):
         )
 
     def cleanup(self):
+        # type: (...) -> None
```

Comment:
```suggestion
        # type: () -> None
```

The `...` isn't needed here.